src/components/pages/ItemDetail/ItemDetail.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import productRequests from '../../../helpers/data/productRequests';
+import './ItemDetail.scss';
+
+class ItemDetail extends React.Component {
+  state = {
+    singleItem: {},
+  };
+
+  backToItemsView = (e) => {
+    this.props.history.push('/store');
+  };
+
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    productRequests.getSingleProductClick(id).then((singleItem) => {
+      this.setState({ singleItem });
+    });
+  }
+
+  render() {
+    const { singleItem } = this.state;
+
+    return (
+      <div className="item-detail mx-auto w-50">
+        <div className="col-9 mt-3 mx-auto">
+          <img className="detailImage" src={singleItem.imageUrl} alt={singleItem.name} />
+          <h4 className="product-card-header">{singleItem.name}</h4>
+          <div className="card-body">
```

Comment:
put a `cursor: pointer` on the card body in .scss so the mouse changes and knows you can click to go to the detail page.